projects/plugins/jetpack/_inc/client/state/products/reducer.js

```diff
@@ -60,5 +60,5 @@ export function isFetchingProducts( state ) {
  * @return {Object}  Products
  */
 export function getProducts( state ) {
-	return state.jetpack.products.items;
+	return get( state.jetpack, [ 'products', 'items' ], {} );
```

Comment:
At one point I was getting a null exception from this when it was accessed before it was initialized, but it's actually not being used anymore since I made a dedicated endpoint to get the upsell info. So I've reverted this change.